scripts-components/toggler/src/index.ts

```diff
@@ -0,0 +1,48 @@
+import { Component, EventListeners, DelegateEvent } from '@mangoweb/scripts-base'
+
+// data-target
+// data-class-name
+// data-toggle-others
+// data-prevent-default
+
+export default class Toggler extends Component<void> {
+	static componentName = 'Toggler'
+
+	getListeners = (): EventListeners => [['click', '.toggler', this.handleClick]]
+
+	handleClick(e: DelegateEvent<'click'>) {
+		const delegateTarget = e.delegateTarget as HTMLElement
```

Comment:
Nemají ty typings pravdu? Vždyť to opravdu může být `SVGElement`, ne? Musel bys dole zaměnit `innerText` za třeba `innerHTML`, ale to v praxi stejně asi bude fungovat identicky.